src/Components/Navbar.js

```diff
@@ -15,16 +16,16 @@ export default function Navbar() {
           <div className="navbar-nav">
             <ul className="navbar-nav ml-auto">
               <li className="nav-item">
-                <Link className="nav-link active" to="/">Quote</Link>
+                <Link className="nav-link active fs-5" to="/">Books</Link>
               </li>
               <li className="nav-item">
-                <Link className="nav-link" to="/categories">Categories</Link>
+                <Link className="nav-link fs-5" to="/categories">Categories</Link>
               </li>
             </ul>
           </div>
         </div>
-        <div className="d-none d-md-block">
-          right-img
+        <div className="d-none d-md-block border border-secondary rounded-circle profile">
+          <div className="text-primary text-center fs-3"><BsFillPersonFill /></div>
```

Comment:
Nice job styling your project, but let's put in more effort to ensure the Nav section matches the design as this is a requirement for this milestone😉.

yours
<img width="1432" alt="image" src="https://github.com/Innocent259/book-store/assets/64643995/c198a47e-da0e-4271-bd0b-5d7dc3b23938">

design
<img width="840" alt="image" src="https://github.com/Innocent259/book-store/assets/64643995/de9e3bec-94f0-4913-950c-9cc4df76fd31">

Take note of the font properties and the spacings and make the changes where necessary😉😉.